src/lib/Scenes/Collection/Screens/CollectionArtworkPreview.tsx

```diff
@@ -0,0 +1,43 @@
+import { Separator } from "@artsy/palette"
+import { CollectionArtworkPreview_collection } from "__generated__/CollectionArtworkPreview_collection.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import { get } from "lib/utils/get"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface Props {
+  collection: CollectionArtworkPreview_collection
+}
+
+export class CollectionArtworkPreview extends React.Component<Props> {
+  render() {
+    let artworks
+    const { collection } = this.props
+    if (!collection) {
+      return null
+    }
+    const collectionArtworks = get(collection.artworks, artwork => artwork.edges, [])
```

Comment:
Does the `get` helper err out if the first argument is undefined? If so, or if you don't feel like looking, a quick way to avoid that happening here would be to move `.artworks` to the second argument, like this:

```suggestion
    const collectionArtworks = get(collection, collection => collection.artworks.artwork.edges, [])
```